gatsby/content/projects/clients/gotktrix.mdx

```diff
@@ -0,0 +1,56 @@
+---
+layout: projectimage
+title: gotktrix
+categories:
+ - client
+thumbnail: /docs/projects/images/gotktrix.png
+description: Matrix client in Go and GTK4.
+author: diamondburned
+maturity: alpha
+language: Go
+license: AGPL-3.0-or-later
+repo: https://github.com/diamondburned/gotktrix
+screenshot: /docs/projects/images/gotktrix.png
+slug: gotktrix
+featured: true
+platforms:
+    - Linux
+    - Mac
+	- Windows
```

Comment:
Github is showing me a large red block here. I suppose this is a tab instead of spaces. Can you look into it?

Reply:
(CI failure confirms something is wrong here)